frontend/src/pages/modelServing/screens/metrics/MetricsChart.tsx

```diff
@@ -108,7 +108,7 @@ const MetricsChart: React.FC<MetricsChartProps> = ({ title, color, metrics, unit
                 </>
               ) : (
                 <>
-                  <EmptyStateIcon variant="container" component={Spinner} />
+                  <EmptyStateIcon icon={Spinner} />
```

Comment:
Didn't Empty state get a header/footer? Was this missed?

Reply:
Yes - nice catch @andrewballantyne looks like the codemods didn't pick up this instance